internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package application_snapshot_image
```

Comment:
This is really awkwardly named, we are validating images which are provided to us via a thing called "ApplicationSnapshot"; or via a simple image reference. This should be simply "image".

Reply:
Can we create some stories to clean it up later?